doc/rados/operations/user-management.rst

```diff
@@ -337,110 +337,104 @@ Pool
 
 A pool is a logical partition where users store data.
 In Ceph deployments, it is common to create a pool as a logical partition for
-similar types of data. For example, when deploying Ceph as a backend for
+similar types of data. For example, when deploying Ceph as a back end for
 OpenStack, a typical deployment would have pools for volumes, images, backups
-and virtual machines, and users such as ``client.glance``, ``client.cinder``,
-etc.
+and virtual machines, and such users as ``client.glance`` and ``client.cinder``.
 
 Application Tags
 ----------------
 
 Access may be restricted to specific pools as defined by their application
 metadata. The ``*`` wildcard may be used for the ``key`` argument, the
-``value`` argument, or both. ``all`` is a synonym for ``*``.
+``value`` argument, or both. The ``all`` tag is a synonym for ``*``.
 
 Namespace
 ---------
 
-Objects within a pool can be associated to a namespace--a logical group of
+Objects within a pool can be associated to a namespace: that is, to a logical group of
 objects within the pool. A user's access to a pool can be associated with a
-namespace such that reads and writes by the user take place only within the
-namespace. Objects written to a namespace within the pool can only be accessed
+namespace so that reads and writes by the user can take place only within the
+namespace. Objects written to a namespace within the pool can be accessed only
 by users who have access to the namespace.
 
 .. note:: Namespaces are primarily useful for applications written on top of
-   ``librados`` where the logical grouping can alleviate the need to create
-   different pools. Ceph Object Gateway (in releases beginning with
-   Luminous) uses namespaces for various
-   metadata objects.
-
-The rationale for namespaces is that pools can be a computationally expensive
-method of segregating data sets for the purposes of authorizing separate sets
-of users. For example, a pool should have ~100 placement groups per OSD. So an
-exemplary cluster with 1000 OSDs would have 100,000 placement groups for one
-pool. Each pool would create another 100,000 placement groups in the exemplary
+   ``librados``. In such situations, the logical grouping provided by namespaces  can obviate the need to create
+   different pools. In Luminous and later releases, Ceph Object Gateway
+   uses namespaces for various metadata objects.
+
+The rationale for namespaces is this: pools can be a computationally expensive
+method of segregating data sets between different authorized users, and namespaces are relatively less computationally expensive.
+For example, a pool ought to have approximately 100 placement groups per OSD. 
+This means that a cluster with 1000 OSDs would have 100,000 placement groups for a single
```

Comment:
This arithmetic is not correct, it is admittedly subtle.

Each *OSD* according to Sage's (misguided) retconned guidance should host ~100 PGs, or more precisely, PG *replicas*.

Say you have 1000 OSDs and three 3R replicated pools.

1000 OSDs x ~100 PG replicas per == 100,000 PG replicas == 33,333 PGs.  Divided by three pools, that means 11,111 per pool.  Bump that to the power of 2, that means 16,384 PGs per pool.

With EC pools, the number of PG replicas (shards) is K+M.